### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by leon0399** [tymondesigns/jwt-auth#2114]
Library seems to be more and more abandoned, there is a lot of pendong PRs and the last contribution was made back in January.

**Comment by CodeNinja1337** [comment]
I'm curious to this as well. Maybe @tymondesigns can give us an answer on this?

I like the library an used it in previous projects but I'm not sure if I should implement it in new projects as well as i don't preffer using unsupported/deprecated libraries. 

(no official support for PHP8.x and Laravel 8.x is a killer for me)

**Comment by specialtactics** [comment]
Tymon generally does quite a bit of work on it, possibly he's a bit busy.

It may be good to appoint some community team members who could manage PRs.

**Comment by CodeNinja1337** [comment]
@specialtactics, I agree that Tymon did/does a lot of work on this and its a great library but there are some feature request that definitely makes sense and it seems that Tymon wasn't active for a while so the question is justly in my opinion. As you mention there are also a lot of open PR's that can be "processed". Unfortunately I'm not conveniënt enough with Laravel to be able to pass a judgement on it else i would like to help a bit.

**Comment by specialtactics** [comment]
Yeh I agree, in this situation and for a package this popular, ideally there should be a team of people so if one person becomes unavailable for some period, the others can hopefully pick up the slack :+1:

**Comment by vicenterusso** [comment]
Already watching closely the repo, and using a fork to support PHP8/L8

**Comment by leon0399** [comment]
I would love to see this project maintained, since it's so popular. It's okay for being busy, we all know what it is. But since project became so popular, it made maintaining this project very important thing.

At this point, what I think, is that we all just have to ask @tymondesigns to find some mainteiners to help. I sure, there is a lot of great developers, who would be really welcome to help supporting this awesome package. But if @tymondesigns would not be able to help us with this search, we might ask @auth0 to help us. Since this project is supported by them, they might help us contacting Tymon, or might be responsible enough to help finding great maintainers and support further development

In the worst scenario, somebody would need to create a well-maintainer fork, but I hope, it will not happen

**Comment by leon0399** [comment]
Hi everyone! Happy to tell, you, there is now a great fork with a lot of issues fixed: https://github.com/PHP-Open-Source-Saver/jwt-auth
